recipe/meta.yaml

```
@@ -1,9 +1,9 @@
 {% set name = "google-api-core" %}
-{% set version = "2.10.1" %}
-{% set sha256 = "e16c15a11789bc5a3457afb2818a3540a03f341e6e710d7f9bbf6cde2ef4a7c8" %}
+{% set version = "2.19.1" %}
+{% set sha256 = "f4695f1e3650b316a795108a76a1c416e6afb036199d1c1f1f110916df479ffd" %}
 
 package:
-  name: {{ name }}
+  name: {{ name }}-split
```

```suggestion
  name: {{ name }}-split
```
Does this split the name by the hyphen?

---

The split has no specific meaning, but giving a different name than the main output changes how conda build interprets the recipe. It doesn't change much in this case though. But that's a way to please the linter :)

---

🙃